tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -70,6 +69,8 @@ Select Project
         Open Project Options Menu
         Click Element    xpath://li/button[.="${project_name}"]
         Wait Until Page Contains Element    //span[.="Project"]/../../..//span[.="${project_name}"]
+        # modal refresh is unreliable
+        Sleep    0.5s
```

Comment:
you could try with the kw `Wait Until Generic Modal Disappears` (Common.robot)

Reply:
I don't need it to disappear, but it refreshes once the project is selected. If execution continues too fast some elements might not be found because they are still (re)loading

Reply:
oh okay right, forget about that then